I2C/I2C.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * I2C.h
+ *
+ *  Created on: Mar 16, 2018
+ *      Author: Phong
+ */
+
+#ifndef I2C_I2C_H_
+#define I2C_I2C_H_
+
+#include "../include.h"
+
+/*
+ * Select I2C module (ex: I2C0)
+ * Enable fast mode (ex: #define FAST_MODE true)
+ */
+#define I2C1
+#define FAST_MODE false
+
+void I2C_Config(void);
```

Comment:
Why don't you use the extern ?

Reply:
Thank you.
I've added "extern"(s) in the [6th commit](https://github.com/VGUHelloWorld/AutoBalanceBot/pull/6/commits/6d1fc8f482e91a9bed4614a0f82972fbca0f4f94).